units/src/fee_rate/serde.rs

```diff
@@ -0,0 +1,254 @@
+// SPDX-License-Identifier: CC0-1.0
+
+// Module implements standardized serde-specific trait methods.
+#![allow(missing_docs)]
+
+//! This module adds serde serialization and deserialization support for amounts.
+//!
+//! Since there is not a default way to serialize and deserialize Amounts, multiple
+//! ways are supported and it's up to the user to decide which serialiation to use.
+//!
+//! The provided modules can be used as follows:
+//!
+//! ```
+//! use serde::{Serialize, Deserialize};
+//! use bitcoin_units::FeeRate;
+//!
+//! #[derive(Serialize, Deserialize)]
+//! pub struct Foo {
+//!     #[serde(with = "bitcoin_units::fee_rate::serde::as_sat_per_kwu")]
```

Comment:
I think ideally if you did not include this, it would default to `sat_per_kwu`.

Reply:
Noted.